kansha/board/boardsmanager.py

```diff
@@ -59,15 +60,23 @@ def _get_board(self, data_board):
         return self._services(Board, data_board.id, self.app_title, self.app_banner, self.theme,
                               self.card_extensions, self.search_engine)
 
+    def create_template_empty(self):
+        '''Get a default empty template'''
+        board = DataBoard(title=u'Empty board', is_template=True)
+        board.visibility = BOARD_PUBLIC
+        for i, (title, color) in enumerate(DEFAULT_LABELS):
```

Comment:
At this point, there is no way to create a label, so you won't be able to use labels in that case.